go.mod

```diff
@@ -4,12 +4,13 @@ go 1.16
 
 require (
 	github.com/google/go-querystring v1.1.0 // indirect
-	github.com/hashicorp/go-retryablehttp v0.6.8 // indirect
+	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
+	github.com/hashicorp/go-retryablehttp v0.7.0 // indirect
 	github.com/hashicorp/go-slug v0.7.0 // indirect
-	github.com/hashicorp/go-tfe v0.15.0
+	github.com/hashicorp/go-tfe v0.16.1-0.20210615172908-c8e00ce06256
```

Comment:
This can be updated to use [`v0.17.0`](https://github.com/hashicorp/go-tfe/releases/tag/v0.17.0)!